includes/transformer/class-post.php

```diff
@@ -302,7 +297,7 @@ protected function get_attachment() {
 		$id    = $this->item->ID;
 
 		// List post thumbnail first if this post has one.
-		if ( \function_exists( 'has_post_thumbnail' ) && \has_post_thumbnail( $id ) ) {
```

Comment:
@pfefferle Do you remember why this check was necessary? Does this code run in an environment where that function might not exist?

Reply:
I think we can remove this check